README.md

```diff
@@ -24,13 +26,17 @@ Please note, we only support UNIX-based systems, and have only tested on macOS,
 
 Install [homebrew](brew.sh) by running the following command:
 
-    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```shell script
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
 
 #### Installing dependencies through Homebrew
 
 Install pkg-config, boost, fftw, nlohmann-json, and cmake by running the following command:
 
-    brew install libomp pkg-config boost fftw nlohmann-json cmake cairo
+```shell script
+brew install libomp pkg-config boost fftw nlohmann-json cmake cairo
```

Comment:
I feel `nlohmann-json` should be added to our repo's `external` folder as third party, and can be removed from here.

Reply:
I don't think that is a good idea, because then we would have even more extra files. Let's us cross that bridge when nlohmann fails us.